reuse.html

```diff
@@ -123,9 +123,9 @@
 </pre>
 </div>
 <p>The <code>header</code> and <code>footer</code> names represent the name of a template (or
-partial) that need to be loaded. If these templates don't exist, the helper body is used.</p>
+partial) that need to be loaded. If these templates don't exist, the original block body is used.</p>
 <p>In the example above <code>header</code> and <code>footer</code> don't exist, that is why we get
 the default content of each of this partials.</p>
 <p>You can find more information <a href="http://thejohnfreeman.com/blog/2012/03/23/template-inheritance-for-handlebars.html">here</a></p>
```

Comment:
delete thes paragraph and link. It was about the old way of doing it but also isn't available anymore